crates/bootstrap_client/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "kitsune2_bootstrap_client"
+version.workspace = true
+description = "p2p / dht communication WAN discovery bootstrapping client"
+license.workspace = true
+homepage.workspace = true
+documentation = "https://docs.rs/kitsune2_bootstrap_client"
+authors.workspace = true
+keywords.workspace = true
+categories.workspace = true
+edition.workspace = true
+
+[dependencies]
+kitsune2_api = { workspace = true }
+
+tracing = { workspace = true }
+ureq = { workspace = true, features = ["tls"] }
+
+[dev-dependencies]
+kitsune2_core = { workspace = true }
+kitsune2_test_utils = { workspace = true }
+kitsune2_bootstrap_srv = { workspace = true }
+
+[features]
```

Comment:
Perhaps a comment can be added here to indicate which features will be added in the future if this is already known

Reply:
Oh good point, I copy-pasted this Cargo.toml and didn't tidy up properly. This can just be removed. I don't think we need any features for now

Reply:
Done: https://github.com/holochain/kitsune2/pull/129/commits/531a406292a63bb0d27852900ca937266eec7d21